lib/class-wp-rest-block-types-controller.php

```diff
@@ -449,6 +451,27 @@ public function get_item_schema() {
 					'context'     => array( 'embed', 'view', 'edit' ),
 					'readonly'    => true,
 				),
+				'context'          => array(
```

Comment:
this can not be called context. It conflicts with existing naming in the API.

Reply:
Yes, I will update it tomorrow, I'm thinking about `use_context` and `provides_context` pair.

Reply:
I renamed `context` property to `use_context` in the REST API endpoint. It's still referenced as `context` inside the `WP_Block_Type` object and everywhere else.